web/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ["google", "plugin:prettier/recommended"],
+  parserOptions: {},
+  plugins: [],
+  rules: {
+    "require-jsdoc": [
+      "error",
+      {
+        require: {
+          FunctionDeclaration: true,
+          MethodDefinition: false,
+          ClassDeclaration: false,
+          ArrowFunctionExpression: false,
+          FunctionExpression: false,
+        },
+      },
+    ],
+  },
```

Comment:
```suggestion
    "quotes": [
        "error",
        "single",
        {
          "avoidEscape": true
        }
      ]
  },
```

Not married to this, so can do without as well 🙈

Reply:
I propose we accept prettier's defaults and diverge only when there's a good reasons. So leaving this off is fine with me.